docs/trace/getting-started/README.md

```diff
@@ -25,45 +25,22 @@ package:
 dotnet add package OpenTelemetry.Exporter.Console -v 0.4.0-beta.2
 ```
 
-Update the `Program.cs` file with the following code:
-
-```csharp
-using System.Diagnostics;
-using OpenTelemetry;
-using OpenTelemetry.Trace;
-
-public class Program
-{
-    private static readonly ActivitySource MyActivitySource = new ActivitySource(
-        "MyCompany.MyProduct.MyLibrary");
-
-    public static void Main()
-    {
-        using var otel = Sdk.CreateTracerProvider(b => b
-            .AddActivitySource("MyCompany.MyProduct.MyLibrary")
-            .UseConsoleExporter());
-
-        using (var activity = MyActivitySource.StartActivity("SayHello"))
-        {
-            activity?.SetTag("foo", 1);
-            activity?.SetTag("bar", "Hello, World!");
-        }
-    }
-}
-```
+Update the `Program.cs` file with the code from Program.cs:
```

Comment:
should we instead modify this entire instructions to:
run this project using `dotnet run`.

And explanation of what this program does.

?

Reply:
Definitely makes sense to me to not have code spread throughout markdown files. It's bound to get out of sync. You also plan to get rid of the sample output? That'll also get out sync.

Reply:
For the getting started in 5 minutes one, I am okay to keep the sample output. Atleast it helps a totally new user to confirm what he see when he runs the sample is the expected behaviour..

Reply:
These are the changes I think we will need to make before GA:
1. Move the `*.md` files to individual folder, for example `docs/trace/getting-started/README.md`.
2. Document has to reflect the latest code. Ideally we can make this automated, or at least have a CI check.
3. Have a way to change the project reference to package reference.